app/code/Magento/Eav/Setup/EavSetup.php

```diff
@@ -90,23 +95,28 @@ class EavSetup
     /**
      * Init
      *
-     * @param ModuleDataSetupInterface $setup
-     * @param Context $context
-     * @param CacheInterface $cache
-     * @param CollectionFactory $attrGroupCollectionFactory
-     * @param Code|null $attributeCodeValidator
+     * @param ModuleDataSetupInterface                     $setup
+     * @param Context                                      $context
```

Comment:
Please discard such changes. 
Use one space between parameter type and parameter name.
Like ` * @param Context $context`

Reply:
I do not agree, but ok: https://devdocs.magento.com/guides/v2.3/coding-standards/docblock-standard-general.html#format-consistency